### PR TITLE
[DOC] Add minimum working examples to clean_names and case_when

### PR DESCRIPTION
**Pull request by ericmjl**
This PR adds minimum working examples to two functions, `clean_names` and `case_when`.

Related to: #972.

### CODE REVIEW

**Comment by ericmjl** [comment]
🚀 Deployed on https://deploy-preview-978--pyjanitor.netlify.app

File: janitor/functions/case_when.py [hunk 1]
```diff
@@ -10,10 +10,40 @@
...
+
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df = pd.DataFrame(
+        ...         {
+        ...             "a": [0, 0, 1, 2, "hi"],
+        ...             "b": [0, 3, 4, 5, "bye"],
+        ...             "c": [6, 7, 8, 9, "wait"],
+        ...         }
+        ...     )
```
Comment (thatlittleboy) [inline janitor/functions/case_when.py]: The indentation for the braces and the final parenthesis are off.

File: janitor/functions/case_when.py [hunk 2]
```diff
@@ -10,10 +10,40 @@
...
+        0   0    0     6     x
+        1   0    3     7     0
+        2   1    4     8     8
+        3   2    5     9     9
+        4  hi  bye  wait    hi
+
+    Similar to SQL and dplyr's case_when
     with inspiration from `pydatatable` if_else function.
 
     If your scenario requires direct replacement of values,
```
Comment (thatlittleboy) [inline janitor/functions/case_when.py:49]: Hmm, Github doesn't allow reviews to be marked on unchanged code as of now 😅

Anyway, just wanted to point out:
- there's a typo of "callabe" at approximately L104 (params docstring for *args).
- and it bugs me a little that in the docstring of `case_when`, `df` is described as "Pandas dataframe" while the `returns` is described as "pandas DataFrame", shall we standardize the casing across the documentation? I've been using the latter in my previous PR.

**Review by thatlittleboy — APPROVED** [review]
I've just pointed a couple formatting issues to fix. The overall logic and "minimalism" of the examples LGTM :)

File: janitor/functions/case_when.py (hunk 1 above)
Comment (ericmjl) [reply janitor/functions/case_when.py]: Good catch! I will push a fix shortly.

File: janitor/functions/case_when.py (hunk 2 above)
Comment (ericmjl) [reply janitor/functions/case_when.py:49]: Good catch on the typo! Thank you for that.

Also, I agree with you about standardizing the casing. We'll go with the latter.

**Pull request merged**